experiments/SlabLand/README.md

```diff
@@ -0,0 +1,156 @@
+- Latent heat flux at surface
+    $$LH = \lambda g_w (q_{sat}(T_{sfc},p_{sfc}) - q_a)$$
+- Downward-pointing short-wave radiative flux
+    $$R_{SW} = F_{sfc}^d - F_{sfc}^u = (1-\alpha)\tau F_{sol} * (1 + \sin(2π τ_d^{-1} t))$$
+- Upward-pointing net long-wave flux
+    $$R_{LW} = F_{sfc}^u - F_{sfc}^d = \epsilon \sigma \theta_{sfc}^4 - \epsilon F_{a}$$
+- Soil storage
+    $$G = \kappa_s (\theta_{sfc}-\theta_h)/h_s$$
+
+For description and values of the parameters above, please see [parameters_initialconditions.jl](parameters_initialconditions.jl). 
+
+### 2. Implemetation
+- for the slab land, we use an explicit time stepping:
+$$T_{sfc}^{n+1} = T_{sfc}^n +\frac{(F_{tot}^{n} + G^{n})}{\rho_s c_s h_s}\Delta t$$
+- this is implemented
+
+        @inline function source!(
+                model::SlabLandModelSetup, source::Vars, state::Vars, gradflux::Vars, aux::Vars, _...)
+            p = model.parameters
+            G    = p.κ_s * (state.T_sfc - p.T_h) / p.h_s # simple soil physics
+            source.T_sfc = - (aux.F_ρθ_prescribed + G) / (p.ρ_s * p.c_s * p.h_s)
+        end
+
+## **Coupler**
+
+### 1. Formulation
+- we're using our sequential coupler, which is described here
+### 2. Surface Flux Handling
+1. Flux calculation
+    - function `calculate_land_sfc_fluxes(_...)` is defined in `coupler/surface_fluxes.jl`, and is used for:
+        - the Atmos Robin boundary condition calculation (see above)
+        - for surface flux accumulation (next point). 
+    - it is called by Atmos, which is assumed to be the model with the shortest timestep. This means that the fluxes are only calculated once and are consistent with Atmos's time stepping.
+
+2. Flux accumulation
+    - In order to integrate fluxes in time consistently with the Atmos time stepper, we define a new prognostic variable `F_ρθ_accum` whose source function accumulates the fluxes, so that $F\_ρθ\_accum = \int F_{tot} dt$
+
+            @inline function source!(atmos_model::ModelSetup, _...)
+                source.F_ρθ_accum = calculate_land_sfc_fluxes(atmos_model, _...) 
+            end
+
+3. Flux / state storage and communication
+    - the coupler stores 2 fields:
+        - `EnergyFluxAtmos` which collects `F_ρθ_accum / Δt` calculated and accumulated in Atmos (with $Δt$ denoting the period of the coupling cycle), and is called by land for its $T_{sfc}$ equation
+        - `LandSurfaceTemperature` which collects the state of the slab Land model, namely $T_{sfc}$ and is called by atmos when calculating surface fluxes
+
+
+## **Tests**
+
+- Dry heat diffusion
+    - Simplifications
+        - $g_a$ is constant
+        - (μ, ν, $\kappa$) = (0, 0, 1e-5)
+        - $\vec{u} = 0$
+        - $G = 0$ and $F_a = 0$
+    - [results](https://docs.google.com/document/d/1JKK8wFKPq3Jo3D4flXZiY3WAUPqYE19pXRvwJgpDwjw/edit)
+
+ (constant conductances and ConstantViscosity( FT(0), FT(0), FT(1e-5), u = 0 )
+
+## **Pipeline** (probably delete)
```

Comment:
I think this pipeline could be worth holding onto for a little while - it's a clear walkthrough of what is happening during the coupling.